.github/workflows/ci.yml

```diff
@@ -246,53 +246,9 @@ jobs:
             deno --version
           fi
 
-      - name: Cache Cargo home
-        uses: actions/cache@v2
-        with:
-          # See https://doc.rust-lang.org/cargo/guide/cargo-home.html#caching-the-cargo-home-in-ci
-          path: |
-            ~/.cargo/registry/index
-            ~/.cargo/registry/cache
-            ~/.cargo/git/db
-          key: g-cargo-home-${{ matrix.os }}-${{ hashFiles('Cargo.lock') }}
-
-      # In main branch, always creates fresh cache
-      - name: Cache build output (main)
-        # TODO(kt3k): Change the version to the released version
-        # when https://github.com/actions/cache/pull/489 (or 571) is merged.
-        uses: actions/cache@03e00da99d75a2204924908e1cca7902cafce66b
-        if: matrix.profile == 'release' && github.ref == 'refs/heads/main'
-        with:
-          path: |
-            ./target
-            !./target/*/gn_out
-            !./target/*/*.zip
-            !./target/*/*.tar.gz
-          key: |
-            g-cargo-target-${{ matrix.os }}-${{ matrix.profile }}-${{ github.sha }}
-
-      # Restore cache from the latest 'main' branch build.
-      - name: Cache build output (PR)
-        # TODO(kt3k): Change the version to the released version
-        # when https://github.com/actions/cache/pull/489 (or 571) is merged.
-        uses: actions/cache@03e00da99d75a2204924908e1cca7902cafce66b
-        if: github.ref != 'refs/heads/main'
-        with:
-          path: |
-            ./target
-            !./target/*/gn_out
-            !./target/*/*.zip
-            !./target/*/*.tar.gz
-          key: never_saved
-          restore-keys: |
-            g-cargo-target-${{ matrix.os }}-${{ matrix.profile }}-
-
-      # Don't save cache after building PRs or branches other than 'main'.
-      - name: Skip save cache (PR)
-        run: echo "CACHE_SKIP_SAVE=true" >> $GITHUB_ENV
-        shell: bash
-        if: github.ref != 'refs/heads/main'
-
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v1
```

Comment:
I'm not sure why, but when it goes to post the cache it says:

```
Post job cleanup.
Warning:  The "path" argument must be of type string. Received type undefined
```

Reply:
Not sure either, but it seems to work. I'll push a few more empty commit here to see how it works.